Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -0,0 +1,358 @@
+        url_suffix = '/insync/endpoints/v1/restores/' + str(restore_id)
+        return self._http_request(method='GET', url_suffix=url_suffix, resp_type='response')
+
+    def post_decommission(self, resource_id):
+        url_suffix = 'https://apis.druva.com/insync/endpoints/v1/devices/' + str(resource_id) + '/decommission'
+        return self._http_request(method='POST', url_suffix=url_suffix, resp_type='response')
+
+
+def Druva_ListQuarantineRanges_Command(clientObj):
+    response = clientObj.get_quarantineRanges()
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Active quarantined Ranges', responseJson['quarantineRanges'])
+        outputs = {"Druva.activeQuarantineRanges(val.rangeID == obj.rangeID)": responseJson['quarantineRanges']}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_FindDevice_Command(clientObj, search_string):
+    response = clientObj.get_findDevice(search_string)
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Found Druva Devices', responseJson['resources'])
+        outputs = {"Druva.Resource(val.resourceID == obj.resourceID)": responseJson['resources']}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_QuarantineResource_Command(clientObj, resource_id, resource_type, from_date, to_date):
+    response = clientObj.post_quarantineResource(resource_id, resource_type, from_date, to_date)
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        headers = ['RangeID']
+        readable_output = tableToMarkdown('Resource quarantined successfully', str(responseJson['rangeID']),
+                                          headers=headers)
+        outputs = {"Druva.QuarantinedRangeID": str(responseJson['rangeID'])}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_DeleteQuarantineRange_Command(clientObj, resource_id, range_id):
+    response = clientObj.delete_quarantineRange(resource_id, range_id)
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        headers = ['RangeID']
+        readable_output = tableToMarkdown('Quarantine Range Deleted Successfully', str(range_id), headers=headers)
+        raw_response = responseJson
+        outputs = {"Druva.deletedQuarantineRange(val.rangeID == obj.rangeID)": str(range_id)}
```

Comment:
delete should not return context outputs

Reply:
Requested Changes made.